vmdb/app/models/miq_server/role_management.rb

```diff
@@ -36,6 +38,10 @@ def sync_active_roles
     @active_role_names = self.active_role_names
   end
 
+  def apache_needed?
+    !(active_role_names & ROLES_NEEDING_APACHE).empty?
```

Comment:
LOL, thanks @Fryguy that was broken.

The callers has to do 
`active_role_names.include_any?(*ROLES_NEEDING_APACHE)`

Until this is merged and released:
https://github.com/ManageIQ/more_core_extensions/pull/18